aiu_fms_testing_utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._version import __version__
+
+__all__ = ['__version__']
```

Comment:
@jbusche 
This line failed `ruff format --check`
```suggestion
__all__ = ["__version__"]
```

Reply:
done!